src/libraries/System.Diagnostics.TraceSource/ref/System.Diagnostics.TraceSource.cs

```diff
@@ -10,8 +10,8 @@ namespace System.Diagnostics
     [System.Diagnostics.SwitchLevelAttribute(typeof(bool))]
     public partial class BooleanSwitch : System.Diagnostics.Switch
     {
-        public BooleanSwitch(string displayName, string description) : base (default(string), default(string)) { }
-        public BooleanSwitch(string displayName, string description, string defaultSwitchValue) : base (default(string), default(string)) { }
+        public BooleanSwitch(string displayName, string? description) : base (default(string), default(string)) { }
+        public BooleanSwitch(string displayName, string? description, string defaultSwitchValue) : base (default(string), default(string)) { }
```

Comment:
`defaultSwitchValue` can't be null?  Or we're just asserting it doesn't make sense for it to be null?

Reply:
It can be null for abstract Switch base class, but not for derived ones all of them parsing this value to int or enum and throws on null